.github/scripts/install-slices/install_slices.py

```diff
@@ -280,6 +293,42 @@ def install_slice(slice: str, arch: str, release: str) -> None:
                 res.stderr.rstrip(),
             )
             sys.exit(res.returncode)
+        # Check if the copyright file has been installed with this slice
+        has_copyright[slice_name] = pathlib.Path(
```

Comment:
I think we could change the name to `installed_copyright` because when reading this I thought it was talking about the upstream package having copyright.

Reply:
sure -> 673670e